### PR TITLE
Edit Organisation Button added

### PR DESCRIPTION
**Pull request by ayush3160**
## Description
Edit organisation button added onClicking it will redirect to /organization route.

## Related Issue

#474 

## Screenshots or GIF (In case of UI changes):
Screenshot 2022-10-16 235015

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by Sarfraz-droid — APPROVED** [review]

File: src/components/ProfileBanner/Organization/index.js [hunk 1]
```diff
@@ -105,10 +106,17 @@ export default function Banner({
               <Grid
                 item
                 xs={12}
-                md={3}
+                md={6}
                 container
                 className={classes.buttonContainer}
               >
+                <Link
```
Comment (Sarfraz-droid) [inline src/components/ProfileBanner/Organization/index.js:113]: also add a check.
**Only and only if the organisation belongs to the user, then show this button**

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

**Review by Niweera — DISMISSED** [review]

File: src/components/ProfileBanner/Organization/index.js (hunk 1 above)
Comment (Sarfraz-droid) [reply src/components/ProfileBanner/Organization/index.js:113]: @ayush3160 please have a look at this

**Comment by Niweera** [comment]
@ayush3160 is this pr finalized?

**Comment by Sarfraz-droid** [comment]
@Niweera there are some changes that are required
- Currently the Edit Organization button will show on non-admins as well as admins we have to add a condition where the user has access to the organization

**Comment by Niweera** [comment]
> @Niweera there are some changes that are required
> 
> * Currently the Edit Organization button will show on non-admins as well as admins we have to add a condition where the user has access to the organization

@ayush3160 can you please address this issue?

**Review by Maahi10001 — APPROVED** [review]

**Comment by Sarfraz-droid** [comment]
Closing this PR due to inactivity

**Pull request closed without merging**